_includes/nav_v1.0.html

```diff
@@ -8,13 +8,8 @@
 					GETTING STARTED
 					 -->
 	<li class="navContainer">Getting Started
```

Comment:
Manually created and edited with Textmate. I need to find a tool that will format it reliably. Textmate at one point reordered and changed all the nesting. I plan to continue working on this file and will clean up the formatting soon. I also have a to-do to add comments to all the javascript.